Doc/c-api/file.rst

```diff
@@ -60,6 +60,27 @@ the :mod:`io` APIs instead.
    raised if the end of the file is reached immediately.
 
 
+.. c:function:: int PyFile_SetOpenCodeHook(void *handler)
```

Comment:
Would it be possible to use a function pointer rather than void* here? void* may cause troubles with Control Flow Integrity (CFI).